.github/workflows/build.yml

```diff
@@ -128,4 +128,15 @@ jobs:
           push: true
           context: .
           tags: ghcr.io/${{ github.repository }}:${{ needs.metadata.outputs.release_version }}
-          labels: com.exactpro.th2.${{ steps.meta.outputs.REPOSITORY_NAME }}=${{ needs.metadata.outputs.release_version }}
\ No newline at end of file
+          labels: com.exactpro.th2.${{ steps.meta.outputs.REPOSITORY_NAME }}=${{ needs.metadata.outputs.release_version }}
+  owasp-scan-job:
+    runs-on: ubuntu-20.04
```

Comment:
You can reuse job from our corporate workflows. There were changes to support multiprojects